opencti-platform/opencti-graphql/src/domain/stixCyberObservable.js

```diff
@@ -282,6 +280,10 @@ export const stixCyberObservableEditField = async (context, user, stixCyberObser
     }
     throw FunctionalError('Cannot update payload_bin when url is present.');
   }
+  if (input[0].key === 'x_opencti_score') {
```

Comment:
Could you iterate on input in case some people use graphQL api with several updates in one call ?

Reply:
done